languages/swift/exercises/concept/tuples/.meta/Example.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+func cartesianToPolar(_ cart: (x: Double, y: Double)) -> (r: Double, theta: Double) {
+  (r: (cart.x * cart.x + cart.y * cart.y).squareRoot(), theta: atan2(cart.y, cart.x))
```

Comment:
Is it possible in Swift to omit the `return` keyword? (would be another similarity to Scala) ;)